cloner.go

```diff
@@ -35,6 +36,10 @@ func (rc *AsyncRepoCloner) Clone(auth transport.AuthMethod) <-chan struct{} {
 		if err != nil {
 			rc.Error = err
 		} else {
+			err := cleanNewRepo(repository)
+			if err != nil {
+				rc.Error = fmt.Errorf("unable to clean new repo: %v", err)
```

Comment:
Do we want to continue on error here? (especially since we set `Ready` to true later)